addons/web/static/src/core/datetime/datetime_picker.xml

```diff
@@ -27,6 +27,7 @@
                             <t t-set="arInfo" t-value="getActiveRangeInfo(itemInfo)" />
                             <button
                                 class="o_date_item_cell o_datetime_button o_center o_cell_md btn p-1 border-0 fw-normal"
+                                tabindex="-1"
```

Comment:
what's the purpose of the `tabindex` on buttons ?

Reply:
tabindex="-1" => not tabable